mtdj/settings/common_api.py

```diff
@@ -13,6 +13,7 @@
     'max_songs_from_category': env.int('MTDJ_SPOTIFY_MAX_SONGS_FROM_CATEGORY', default=25),
     'max_playlist_from_category': env.int('MTDJ_SPOTIFY_MAX_PLAYLISTS_FROM_CATEGORY', default=10),
     'auth_cache_key': env.str('MTDJ_SPOTIFY_AUTH_CACHE_KEY', default='spotify:auth-token'),
+    'auth_redirect_uri': env.str('MTDJ_SPOTIFY_REDIRECT_URI', default='https://moodytunes.vm/moodytunes/spotify/callback/'),
```

Comment:
Will need a corresponding PR to cradle to set the production redirect URI

NOTE: Need to whitelist in the prod app on Spotify

Reply:
Created https://github.com/Moody-Tunes/cradle/pull/62 to add production Spotify OAuth redirect URI